crates/driver/src/infra/config/file/mod.rs

```diff
@@ -92,14 +92,7 @@ struct SubmissionConfig {
 #[serde(rename_all = "kebab-case", deny_unknown_fields)]
 enum Mempool {
     #[serde(rename_all = "kebab-case")]
```

Comment:
The variant no longer contains any fields so the `kebab-case` renaming can be dropped.

Reply:
Fixed.